src/components/Calculator.js

```diff
@@ -0,0 +1,42 @@
+import './Calculator.css';
+
+const CalcInside = (porps) => {
+  const { calcText, colorClass } = porps;
+
+  return (
+    <div className={`wrap ${colorClass}`}>
+      <p>{calcText}</p>
+    </div>
+  );
+};
+
+// f5913e e0e0e0
+
```

Comment:
- [ ] Kindly 😃 try to remove this comment as it makes your codebase ugly, so let's remove it to make our code beautiful and follow the best practice to write clean code 😊💯